zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/OpensearchVersion.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.elasticsearch;
+
+import java.util.Objects;
+
+/** Helps avoid problems comparing versions by number. Ex 2.10 should be > 2.9 */
+public final class OpensearchVersion extends BaseVersion implements Comparable<OpensearchVersion> {
```

Comment:
please unit test this class or the base one?

Reply:
Did both, thanks!